addons/hr_expense/hr_expense_view.xml

```diff
@@ -1,293 +1,399 @@
+        <field name="action_id" ref="action_run_expense_sheet" />
+        <field name="value" eval="'ir.actions.server,' + str(ref('action_run_expense_sheet'))" />
+        <field name="key">action</field>
+        <field name="model_id" ref="model_hr_expense_sheet" />
+        <field name="model">hr_expense.sheet</field>
+        <field name="key2">client_action_multi</field>
+    </record>
+    
+    <record id="hr_expense_product" model="ir.actions.act_window">
+        <field name="name">Expense Products</field>
+        <field name="res_model">product.product</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">kanban,tree,form</field>
+        <field name="context">{"default_can_be_expensed":1}</field>
+        <field name="domain">[('can_be_expensed','=',True)]</field>
+        <field name="search_view_id" ref="product.product_search_form_view"/>
+        <field name="help" type="html">
+          <p class="oe_view_nocontent_create">
+            Click to create a new expense category. 
+         </p>
+        </field>
+    </record>
 
-        <record id="hr_expense_product" model="ir.actions.act_window">
-            <field name="name">Expense Products</field>
-            <field name="res_model">product.product</field>
-            <field name="view_type">form</field>
-            <field name="view_mode">kanban,tree,form</field>
-            <field name="context">{"default_hr_expense_ok":1}</field>
-            <field name="domain">[('hr_expense_ok','=',True)]</field>
-            <field name="search_view_id" ref="product.product_search_form_view"/>
-            <field name="help" type="html">
-              <p class="oe_view_nocontent_create">
-                Click to create a new expense category.
-             </p>
-            </field>
-        </record>
+    <record id="action_approved_expense" model="ir.actions.act_window">
+        <field name="name">Expenses</field>
+        <field name="res_model">hr_expense.sheet</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+        <field name="view_id" ref="view_expense_to_approve_tree"/>
+        <field name="domain">[('state','=','accepted')]</field> 
+    </record>
+    
+    <record id="action_expense_to_approve" model="ir.actions.act_window">
+        <field name="name">Expenses to Approve</field>
+        <field name="res_model">hr_expense.sheet</field>
+        <field name="view_type">form</field>
+        <field name="context">{'search_default_confirm':1}</field>
+        <field name="search_view_id" ref="view_expense_to_approve_filter"/>
+        <field name="view_id" ref="view_expense_to_approve_tree"/>
+        <field name="help" type="html">
+          <p class="oe_view_nocontent_create">
+            Click to register new expenses. 
+          </p><p>
+            Odoo will ensure the whole process is followed; the expense
```

Comment:
Wording : Once you have created your expense, submit it to your manager who will validate it.